.github/workflows/release.yml

```diff
@@ -100,26 +101,26 @@ jobs:
        
 
     - name: Upload artifact
-      uses: actions/upload-artifact@v1.0.0
+      uses: actions/upload-artifact@v3.1.2
       with:
         name: thunderbird-beta-continuous-x86_64.AppImage
         path: 'thunderbird/dist'
 
   Thunderbird-Nightly:
-    runs-on: ubuntu-18.04
+    runs-on: ubuntu-20.04
     strategy:
       matrix:
         version: ['3.8']
     steps:
-    - uses: actions/checkout@v2
+    - uses: actions/checkout@v3.5.3
 
     - name: Download Thunderbird Nightly
       run: |
         sudo apt -y install libxml2-utils
         mkdir thunderbird
         cd thunderbird
-        wget https://www.thunderbird.net/en-US/ -O index.html 
-        wget $(xmllint --html index.html --xpath '/html/body/main/section[5]/section/article[2]/section[3]/main/aside/div/ul/li[4]/a' 2>/dev/null | grep -E -o 'href="(.*" )' | cut -d " " -f1 | grep -E -o '"(.*)"' | sed 's/\&amp;/\&/g' | tail -n 1 | sed 's,",,g') -O thunderbird.tar.bz2
+        wget https://www.thunderbird.net/en-US/about -O index.html 
+        wget $(xmllint --html index.html --xpath '/html/body/main/section[4]/section/article[2]/section[3]/main/aside/div/ul/li[4]/a' 2>/dev/null | grep -Po 'href="\K[^"]*' | sed 's|\&amp;|\&|g') -O thunderbird.tar.bz2
```

Comment:
-P flag = Perl compatible (needed for '\K') 
'href="' - matches start of url
'\K' - resets search (grep will not include text before it)
'[^"]*' - matches all characters until double-quote